### ISSUE TITLE
Terminal icon doesn't work

### ISSUE THREAD

**Issue by yume-chan** [Gruntfuggly/activitusbar#38]
I added an icon for terminal in settings.json

```json
"activitusbar.views": [
    {
      "name": "terminal",
    }
  ]
```

Click it once does nothing, and click it twice toggles sidebar.

I also see errors in DevTools:

![image](https://user-images.githubusercontent.com/1330321/101116576-5d790e00-3620-11eb-9e92-539475810191.png)

Looking at keybindings.json, the command name should be `workbench.action.terminal.focus` or `terminal.focus`

Code Version:

```text
Version: 1.52.0-insider (user setup)
Commit: 92192baf760315ac47fb2caaf3ce6d0778962352
Date: 2020-12-03T05:43:59.635Z
Electron: 9.3.5
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.17763
```

**Comment by Gruntfuggly** [comment]
Yes, you're right. Maybe the API changed. You can make it work for now, by using
```
        {
            "name": "command.workbench.action.terminal.focus",
            "codicon": "terminal"
        }
```

I'll fix it properly in the next release.

**Comment by yume-chan** [comment]
That works, thanks!

I will leave this issue open and let you close it after fixed.

**Comment by Gruntfuggly** [comment]
Should be fixed in 0.0.37

**Comment by yume-chan** [comment]
Fix verified.
